apps/vaporgui/VContainer.cpp

```diff
@@ -0,0 +1,49 @@
+#include <QEvent>
+#include <QWidget>
+
+#include "VContainer.h"
+
+VContainer::VContainer( QWidget* containee ) 
```

Comment:
Why does this take `QWidget* containee` but not do anything with it?

Reply:
Good catch.  Fixed.